src/pages/productDetailsPage/ProductDetails.jsx

```diff
@@ -325,14 +332,22 @@ const ProductDetails = () => {
                 />
               </div>
             )}
-          {shouldHaveFbtProductsValue && fbtRecommendationsProducts.length > 0 && (
-            <div>
+          {shouldHaveFbtProductsValue && fbtRecommendationsProducts.length > 1 && (
+            <>
```

Comment:
Because we add the current PDP product into to that array, and we don't want to show FBT if there's just the current product but no recommendations, there will always be one, even if there are no recommendations